server/validation/email.js

```diff
@@ -0,0 +1,11 @@
+const validateEmail = (email) => {
+  if (!email) return false;
+
+  // eslint-disable-next-line no-useless-escape
+  const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+  const isValid = emailRegex.test(email);
+  return isValid;
+};
+
+module.exports = validateEmail;
```

Comment:
I like this... but there are libraries for it. I'm a fan of not using a library, but just for knowledge sake, I think there is a Validator library